### PR TITLE
Open external links in new tab - Closes #522

### LINKED ISSUE #522: Update external links in Core docs
Parent issue #521

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #522

### CODE REVIEW

File: modules/ROOT/pages/interact-with-network.adoc [hunk 1]
```diff
@@ -80,20 +80,20 @@ Please see the list of all commands and their example responses on the following
 [[lisk_desktop]]
 === C - Use Lisk Desktop
 
-{url_lisk_wallet}[Lisk Desktop] is the Graphical User Interface (GUI) to interact with the network.
+{url_lisk_wallet}[Lisk Desktop^] is the Graphical User Interface (GUI) to interact with the network.
```
Comment (chris529) [inline modules/ROOT/pages/interact-with-network.adoc]:
```suggestion
{url_lisk_wallet}[Lisk Desktop^] is the Graphical User Interface (GUI) deployed in oder to interact with the network.
```

File: modules/ROOT/pages/management/binary.adoc [hunk 2]
```diff
@@ -159,7 +159,7 @@ The advanced commands which can be used to manage your Lisk node, can be seen in
 
 The snapshot script is used to take a backup of the whole blockchain.
 A snapshot can be used to speed up the sync process, instead of having to validate all of the transactions starting from block height 0 to the current block height.
-Lisk provides official snapshots of the blockchain, which can be seen here: {url_lisk_snapshots}.
+Lisk provides official snapshots of the blockchain, which can be seen here: {url_lisk_snapshots}[http://snapshots.lisk.io^].
```
Comment (chris529) [inline modules/ROOT/pages/management/binary.adoc]:
```suggestion
Lisk provides official snapshots of the blockchain which can be seen here: {url_lisk_snapshots}[http://snapshots.lisk.io^].
```

File: modules/ROOT/pages/update/source.adoc [hunk 3]
```diff
@@ -36,7 +36,7 @@ npm run build <6>
 
 <1> Navigate into the root directory of Lisk Core.
 <2> Fetch the latest releases.
-<3> Check out latest release tag, please see {url_core_releases}[core releases].
+<3> Check out latest release tag, please see {url_core_releases}[core releases^].
```
Comment (chris529) [inline modules/ROOT/pages/update/source.adoc]:
```suggestion
<3> To view the latest release tag, please see {url_core_releases}[core releases^].
```

File: modules/ROOT/pages/setup/source.adoc [hunk 4]
```diff
@@ -270,7 +270,7 @@ In addition, do not forget to update this password in the xref:{url_core_config}
 
 === PM2 (optional)
 
-Install {url_pm2}[PM2] for managing start/stop of the application process in the background using the following command listed below:
+Install {url_pm2}[PM2^] for managing start/stop of the application process in the background using the following command listed below:
```
Comment (chris529) [inline modules/ROOT/pages/setup/source.adoc]:
```suggestion
Install {url_pm2}[PM2^] for managing both the start/stop function of the application process in the background using the following command listed below:
```

**Review by chris529 — COMMENTED** [review]
Looks very good, only minor points :-)

File: modules/ROOT/pages/interact-with-network.adoc (hunk 1 above)
Comment (Tschakki) [reply modules/ROOT/pages/interact-with-network.adoc]: Is the sentence without this not correct? I would prefer to use as few words as possible.

Also I think the term `deployed` might be a little irritating here,  I would prefer `installed` or similar in this case.

File: modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -134,9 +131,9 @@ Within this network the Lisk users can transfer LSK tokens from one account to a
 The Testnet is an independent replica of the Lisk Mainnet, whose main function is to test the upgrades first before they are run on the Lisk Mainnet.
 Subsequently, this is where new/updated versions and fixes of the Lisk Core are tested.
 
-To start using the Testnet, please download your free LSK Testnet from the {url_faucet_testnet}[Testnet Faucet].
+To start using the Testnet, please download your free LSK Testnet from the {url_faucet_testnet}[Testnet Faucet^].
 
-To connect to Testnet via {url_lisk_desktop}[Lisk Desktop], simply enable the "Network Switcher" in the settings and then use it to switch the network to `Testnet`.
+To connect to Testnet via {url_lisk_desktop}[Lisk Desktop^], simply enable the "Network Switcher" in the settings and then use it to switch the network to `Testnet`.
```
Comment (chris529) [inline modules/ROOT/pages/index.adoc]:
```suggestion
To connect to the Testnet via {url_lisk_desktop}[Lisk Desktop^], simply enable the "Network Switcher" in the settings and then use it to switch the network to `Testnet`.
```

File: modules/ROOT/pages/interact-with-network.adoc (hunk 1 above)
Comment (chris529) [reply modules/ROOT/pages/interact-with-network.adoc]: OK, understood...well the original sentence does not exactly read correctly. We could use the following to keep the sentence as short as possible:
```suggestion
{url_lisk_wallet}[Lisk Desktop^] is the Graphical User Interface (GUI) used to interact with the network.
```

**Review by chris529 — COMMENTED** [review]
Looks very good Mona, please see the suggestion for line no. 83.  :-)

File: modules/ROOT/pages/interact-with-network.adoc (hunk 1 above)
Comment (Tschakki) [reply modules/ROOT/pages/interact-with-network.adoc]: great!

**Review by chris529 — APPROVED** [review]
Perfect :-)

**Pull request merged**